etc/Vagrantfile

```diff
@@ -0,0 +1,20 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.box = "fedora/25-cloud-base"
+  config.vm.box_check_update = true
+
+  config.vm.synced_folder(
+    "../../c-extension-tutorial", "/home/vagrant/c-extension-tutorial"
```

Comment:
I don't know if this is required for vagrant but this is going two directories up and then assuming the clone-directory is called "c-extension-tutorial", could this just be `../`?

If it can be, I  can make the change.